.proto/CreateMediaItems.proto

```
@@ -0,0 +1,61 @@
+syntax = "proto3";
```

Buf (1.72.0)

[error] Files must have a package defined.

(PACKAGE_DEFINED)